### ISSUE TITLE
Remove old code with comment "this stuff should go away"?

### ISSUE THREAD

**Issue by alexislefebvre** [liip/LiipFunctionalTestBundle#251]
I found a surprising comment in `Test/Html5WebTestCase.php` ([example in 1.5 version](https://github.com/liip/LiipFunctionalTestBundle/blob/1.5.0/Test/Html5WebTestCase.php#L156-L160)):

```
/*
 * unfortunately, the bamboo html5 validator.nu gives back an empty "message" about the error with brightcove object, but we have to ignore the error
 * if our local validator.nu instance is fixed, this stuff should go away
 */
$ignores_extract = $this->getContainer()->getParameter('liip_functional_test.html5validation.ignores_extract');
```

Should this code be removed?

It was added 5 years ago: https://github.com/liip/LiipFunctionalTestBundle/commit/a4a919039e07df073c73d4f13eb8e12299e2d824

**Comment by lsmith77** [comment]
yeah .. probably.

**Comment by alexislefebvre** [comment]
Fixed by #375.
